### ISSUE TITLE
Uncaught ReferenceError: iconAndAliases

### ISSUE THREAD

**Issue by Nerexbcd** [lucide-icons/lucide#2105]
### Package

- [X] lucide
- [ ] lucide-angular
- [ ] lucide-flutter
- [ ] lucide-preact
- [ ] lucide-react
- [ ] lucide-react-native
- [ ] lucide-solid
- [ ] lucide-svelte
- [ ] lucide-vue
- [ ] lucide-vue-next
- [ ] Figma plugin
- [ ] source/main
- [ ] other/not relevant

### Version

0.376.0

### Can you reproduce this in the latest version?

- [X] Yes
- [ ] No

### Browser

- [X] Chrome/Chromium
- [ ] Firefox
- [ ] Safari
- [ ] Edge
- [ ] iOS Safari
- [ ] Opera
- [ ] Other/not relevant

### Operating system

- [X] Windows
- [ ] Linux
- [ ] macOS
- [ ] ChromeOS
- [ ] iOS
- [ ] Android
- [ ] Other/not relevant

### Description
Hi,
I am currently developing a web page using lucide icons trough the CDN, earlier today the pages was working fine, but a few minutes ago, when i started working again the icons ware not showing, I checked the console and i was getting this error:
```
Uncaught ReferenceError: iconAndAliases is not defined
    at Object.createIcons (createIcons.ts:7:24)
    at users:133:16
```

### Steps to reproduce

1. Import the js script trough the CDN
2. Try to execute the `createIcons` method

### Checklist

- [X] I have searched if someone has submitted a similar issue before and there weren't any. (Please make sure to also search closed issues, as this issue might already have been resolved.)

**Comment by Nerexbcd** [comment]
The error is appearing due to a missing import, that was caused when @ericfennis, separated the `createIcons` function from the `packages/lucide/src/lucide.ts` file to `packages/lucide/src/createIcons.ts` but didn't copied the folowing line:
```js
import * as iconAndAliases from './iconsAndAliases';
```

**Comment by ericfennis** [comment]
@Nerexbcd Thanks I will make a fix!

**Comment by ericfennis** [comment]
I've reverted the change, it's available on `lucide@0.376.1`

**Comment by stephenbielecki-avt** [comment]
Woo! Thanks for reporting this - noticed the same error and was about to report it. Thanks for getting to it so quickly @ericfennis ! 😄
